src/shared/services/commandInterpreterHelper.js

```diff
@@ -184,24 +184,24 @@ const availableCommands = [
             'No multi db support detected.'
           )
         }
-        // Check if chosen db exists
-        const dbExists = getDatabases(store.getState())
-          .map(db => db.name)
-          .includes(dbName)
-        if (!dbExists) {
+        // Check if chosen db exists. Case insensitive.
+        const existingDb = getDatabases(store.getState()).find(
+          db => db.name.toLowerCase() === dbName.toLowerCase()
```

Comment:
Nitpick: You could use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) here.